src/applications/vaos/project-cheetah/components/SecondDosePage.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import * as actions from '../redux/actions';
+import FormButtons from '../../components/FormButtons';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import moment from 'moment';
+import { getReviewPage } from '../redux/selectors';
+import AdditionalInfoRow from '../../appointment-list/components/cards/AdditionalInfoRow';
+
+const pageKey = 'secondDosePage';
+const pageTitle = 'Plan your second dose';
+
+function goBack({ routeToPreviousAppointmentPage, history }) {
+  return routeToPreviousAppointmentPage(history, pageKey);
+}
+
+function SecondDosePage({
+  data,
+  facility,
+  pageChangeInProgress,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+}) {
+  const [showMoreOpen, setShowMoreOpen] = useState(false);
+  const history = useHistory();
+  const { date1, vaFacility } = data;
+
+  useEffect(() => {
+    if (history && !vaFacility) {
+      history.replace('/new-project-cheetah-booking');
+    }
+    document.title = `${pageTitle} | Veterans Affairs`;
+    scrollAndFocus();
+  }, []);
+
+  return (
+    <div>
+      <h1>{pageTitle}</h1>
+      <div>
+        <p>
+          You'll need to return to the {facility.name} for your second dose.
+          Your team will schedule your second dose after you receive you first.
+        </p>
+        <p>
+          If you receive your first dose on{' '}
+          <strong>
+            {moment(date1, 'YYYY-MM-DDTHH:mm:ssZ').format(
```

Comment:
You should be able to leave off the format string and do `moment(date1).format(...`. Moment expects the standard ISO string format by default.

Reply:
done

Reply:
I see date1 is missing the "Z" after the standard ISO string. Incidentally very topic we discussed during our 1:1 
So for the time being i'm going to leave the format string as is.

Reply:
I had forgotten that `date1` is an array, so if you do `moment(date1[0])`, without the format string, it parses correctly. I'm not really sure why it works with the formatting string, moment seems to be more lenient in that case.

The `Z` is missing on the date in the form data because the date isn't in UTC, which is what `Z` means. That's expected, as the dates we get back for appointment slots don't have a timezone specified and are assumed to be in the local facility time. The `Z` at the end of your formatting string doesn't have an affect on how the date is parsed, as far as I can tell, since it is tell moment to look for a string like +05:00 at the end of the date, which it does not have.

It would probably make sense to have our calendar picker deal with UTC dates to be consistent, but since we control how those dates get used and we know what the backend expects, it doesn't cause problems. For the ICS appointment file it's a problem because calendar apps don't know what timezone an appointment is unless we pass that `Z` to indicate UTC or pass a specific timezone description.

Reply:
Getting the correct timezone and involvement with `Z` is so very confusing. Thanks for the explanation. Accessing  the `date1` as an array worked perfectly.